app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -0,0 +1,321 @@
+import { request } from '../utilities/http/request';
+
+function addCloseListener() {
+  const button = document.querySelector('.close-actions-panel');
+  button.addEventListener('click', () => {
+    // getting the article show page document because this is called within an iframe
+    const articleDocument = window.parent.document;
```

Comment:
You can use `top` instead. See https://developer.mozilla.org/en-US/docs/Web/API/Window/top
```suggestion
    const articleDocument = top.document;
```